src/docs/Glossary.adoc

```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
 [[_glossary_workflow_studio]]
 Workflow Studio:: ProActive component that provides a web interface for designing <<_glossary_workflow,*Workflows*>>.
 
+[[_glossary_automation_dashboard]]
+Automation Dashboard:: Centralized interface for the following web portals: <<_glossary_workflow_automation_portal,*Workflow Automation*>>, <<_glossary_catalog_portal,*Catalog*>>, <<_glossary_analytics_portal,*Analytics*>>, <<_glossary_job_planner_portal,*Job Planner*>>, <<_glossary_cloud_automation_portal,*Cloud Automation*>>, <<_glossary_cloud_watch_portal,*Cloud Watch*>> and <<_glossary_notification_portal,*Notifications*>>.
+
+[[_glossary_analytics]]
+Analytics::  ProActive service responsible to gather monitoring information for <<_glossary_job,*Jobs*>> and <<_glossary_proactive_node,*ProActive Nodes*>>
+
 [[_glossary_job_analytics]]
-Job Analytics:: ProActive component that provides an overview of executed workflows along with their input variables and results.
+Job Analytics:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of executed workflows along with their input variables and results.
+
+[[_glossary_node_gantt]]
+Node Gantt:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of <<_glossary_proactive_node,*ProActive Nodes*>> usage over time.
+
+[[_glossary_analytics_portal]]
+Analytics Portal:: Web interface of the <<_glossary_job_analytics,*Analytics*>> component.
 
 [[_glossary_notification_service]]
 Notification Service:: ProActive component that allows a user or a group of users to subscribe to different types of notifications (web, email, sms or a third-party notification) when certain event(s) occur (e.g., job In-Error state, job in Finished state, scheduler in Paused state, etc.).
 
 [[_glossary_notification_portal]]
-Notification Portal::
-ProActive portal to visualize notifications generated by the <<_glossary_notification_service,*Notification Service*>> and manage subscriptions.
+Notification Portal:: Web interface to visualize notifications generated by the <<_glossary_notification_service,*Notification Service*>> and manage subscriptions.
 
 [[_glossary_notification_subscription]]
-Notification Subscription::
-Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
+Notification Subscription:: Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
 
 [[_glossary_notification_third_party]]
-Third Party Notification::
-Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
+Third Party Notification:: Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
```

Comment:
We can append another sentence to better explain, e.g., :
This script is generally used to send notifications to a third party monitoring system in the customer environment.